Build script is broken

Just found the project, loved it and thought of contributing. 😄 

But after running yarn, the build script is executed and gives two typescript errors:
```
src/utils/throttle.ts(11,7): error TS7034: Variable 'callbackArgs' implicitly has type 'any' in some locations where its type cannot be determined.
src/utils/throttle.ts(15,25): error TS7005: Variable 'callbackArgs' implicitly has an 'any' type.
```

The fix should be as easy as providing type information for the callbackArgs variable. In case the issue is confirmed, I'll be submitting a PR.

--- a/src/utils/throttle.ts
+++ b/src/utils/throttle.ts
@@ -8,7 +8,7 @@
 
 export function throttle(func: Function, wait: Number) {
   let timeout: number | null = null;
-  let callbackArgs = null;
+  let callbackArgs: IArguments | null = null;
   const context = this;
 
   const later = () => {

